src/main/java/de/dagere/peass/ci/logs/measurement/MeasurementActionCreator.java

```diff
@@ -67,11 +67,18 @@ private void createLogActions(final Run<?, ?> run, final Map<TestCase, List<LogF
          TestCase testcase = entry.getKey();
          int vmId = 0;
          for (LogFiles files : entry.getValue()) {
-            String logData = processConfig.getFileText(files.getCurrent());
-            run.addAction(new LogAction(IdHelper.getId(), testcase, vmId, measurementConfig.getFixedCommitConfig().getCommit(), logData));
-            String logDataOld = processConfig.getFileText(files.getPredecessor());
-            run.addAction(new LogAction(IdHelper.getId(), testcase, vmId, measurementConfig.getFixedCommitConfig().getCommitOld(), logDataOld));
+
+            if (files.getCurrent().exists() && files.getPredecessor().exists()) {
```

Comment:
This would not render the files if one of the files exists - could you adapt it, so it at least displays all the files that are available?